default_settings.json

```diff
@@ -0,0 +1,5254 @@
+	"timeline.pageSize": null,
+
+	// Configure settings to be overridden for the css language.
+	"[css]":  {
+		"editor.suggest.insertMode": "replace"
+	},
+
+	// Configure settings to be overridden for the dockercompose language.
+	"[dockercompose]":  {
+		"editor.insertSpaces": true,
+		"editor.tabSize": 2,
+		"editor.autoIndent": "advanced"
+	},
+
+	// Configure settings to be overridden for the dockerfile language.
+	"[dockerfile]":  {
+		"editor.quickSuggestions": {
+				"strings": true
+		}
+	},
+
+	// Configure settings to be overridden for the erb language.
+	"[erb]":  {
+		"editor.formatOnSave": true
+	},
+
+	// Configure settings to be overridden for the gemfile language.
+	"[gemfile]":  {
+		"editor.formatOnSave": true
+	},
+
+	// Configure settings to be overridden for the git-commit language.
+	"[git-commit]":  {
+		"editor.rulers": [
+				72
+		],
+		"workbench.editor.restoreViewState": false
+	},
+
+	// Configure settings to be overridden for the git-rebase language.
+	"[git-rebase]":  {
+		"workbench.editor.restoreViewState": false
+	},
+
+	// Configure settings to be overridden for the go language.
+	"[go]":  {
+		"editor.insertSpaces": false
+	},
+
+	// Configure settings to be overridden for the handlebars language.
+	"[handlebars]":  {
+		"editor.suggest.insertMode": "replace"
+	},
+
+	// Configure settings to be overridden for the html language.
+	"[html]":  {
+		"editor.suggest.insertMode": "replace"
+	},
+
+	// Configure settings to be overridden for the javascript language.
+	"[javascript]":  {
+		"editor.maxTokenizationLineLength": 2500
+	},
+
+	// Configure settings to be overridden for the json language.
+	"[json]":  {
+		"editor.quickSuggestions": {
+				"strings": true
+		},
+		"editor.suggest.insertMode": "replace"
+	},
+
+	// Configure settings to be overridden for the jsonc language.
+	"[jsonc]":  {
+		"editor.quickSuggestions": {
+				"strings": true
+		},
+		"editor.suggest.insertMode": "replace"
+	},
+
+	// Configure settings to be overridden for the less language.
+	"[less]":  {
+		"editor.suggest.insertMode": "replace"
+	},
+
+	// Configure settings to be overridden for the makefile language.
+	"[makefile]":  {
+		"editor.insertSpaces": false
+	},
+
+	// Configure settings to be overridden for the markdown language.
+	"[markdown]":  {
+		"editor.unicodeHighlight.ambiguousCharacters": false,
+		"editor.unicodeHighlight.invisibleCharacters": false,
+		"editor.wordWrap": "on",
+		"editor.quickSuggestions": false
+	},
+
+	// Configure settings to be overridden for the plaintext language.
+	"[plaintext]":  {
+		"editor.unicodeHighlight.ambiguousCharacters": false,
+		"editor.unicodeHighlight.invisibleCharacters": false
+	},
+
+	// Configure settings to be overridden for the ruby language.
+	"[ruby]":  {
+		"editor.formatOnSave": true
```

Comment:
?

Reply:
It looks like this doesn't format on autosave.